.github/workflows/build-windows.yml

```diff
@@ -34,7 +34,7 @@ jobs:
       - name: Add msbuild to PATH
         uses: microsoft/setup-msbuild@v1.0.2
         with:
-          vs-version: '[16.0,16.10)'
+          vs-version: '[16.0,16.30)'
```

Comment:
> I suppose only the newest MSVS installed on CI so I couldn't choose older one.

What does this line do then?

Reply:
Searches any msbuild with that version and add it to `PATH`.

Reply:
I meant what does changing from `.10` to `.30` do?

Reply:
To allow `MSVC 19.29.30037.0` be included.

Reply:
If "only the newest" version is installed, why do you need to specify a version at all?

Reply:
> Or is it the newest version of MSVC2019 and you want to specify not MSVC2017?

Yes.

Reply:
Makes sense then.

Otherwise, as above,
> I don't have any relevant comment...